src/proto/h2/client.rs

```diff
@@ -60,6 +60,7 @@ where
         loop {
             let next = match self.state {
                 State::Handshaking(ref mut h) => {
+                    trace!("Polled client future, state is handshaking");
```

Comment:
I'd probably remove these traces, I'm sure they were useful developing the feature, but afterwards, debugging other things will likely mean these just feel more like noise.